django-verdant/rca/static/rca/js/nav.js

```diff
@@ -161,7 +161,7 @@ var desktopNav = {
 		$('nav').each(function(){
 			$(this).unbind().removeClass('ready').removeClass('changing').attr('style', '');
 			$('li, ul', $(this)).unbind().attr('style', '');
-			$('.breadcrumb', $(this)).remove()
+			//$('.breadcrumb', $(this)).remove()
```

Comment:
Let's delete this line instead.

Reply:
Instead I've removed the call to nav.js altogether as it only relates to the old navigation that no longer exists.